src/frontend/web_application/src/components/MultidimensionalPi/components/Ratings/style.scss

```diff
@@ -43,4 +45,27 @@ $gridColor: $co-color__fg__back--higher;
     padding-right: $co-component__spacing;
     text-transform: uppercase;
   }
+
+  &--mini {
+    max-width: $mini-pi__width;
+    cursor: help;
+
+    .m-pi-ratings__item {
```

Comment:
this should not be formated like this, it actually is what BEM tries to prevent.
Remember, when you see a DOM element with the class `m-pi-ratings--mini__item` and `m-pi-ratings__item` you will know what css directives you will find and what you can change without breaking other element design : there's no side-effects.

Long story short this should noted like this:

```
.m-pi-ratings {
  //...
  &--mini {
    // ...
    &__item {
      // ...
    }
    // ...
}
```

so in the template you have to declare it each time (and it is quite verbose but the safest way to know what happen):

`<item className={classnames('m-pi-ratings__item', 'm-pi-ratings--mini__item')} />`

That's the theory. In real world it is more pragmatic to leave it like this but it will never been fixed.